src/main/java/org/asciidoc/intellij/ui/PasteImageDialog.java

```diff
@@ -108,6 +112,23 @@ public String getSelectedActionCommand() {
     return buttonGroup.getSelection().getActionCommand();
   }
 
+  @Override
+  protected boolean postponeValidation() {
+    return false;
+  }
+
+  @Override
+  protected @Nullable ValidationInfo doValidate() {
```

Comment:
The solution with this hook is really neat. I tried it with adding a custom `ComponentValidator` which was a lot more complex and verbose...